primary_beam_correction.py

```diff
@@ -1,23 +1,11 @@
 #!/usr/bin/env python
-
-"""Input a continuum image from the MeerKAT SDP pipeline and produce a primary beam corrected image.
-Last updated: 12/06/2018
-Usage:
-  primary_beam_correction.py -h | --help
-  primary_beam_correction.py --path=<path>
-
-Try:
-  primary_beam_correction.py --path /path/to/MeerKAT/primary_beam_uncorrected_image
-
-Options:
--h --help                   show the help message"""
-
 import numpy as np
-from docopt import docopt
 from astropy.io import fits
 from astropy import wcs
 from astropy import coordinates
 from astropy import units as u
+import argparse
+import logging
```

Comment:
System imports should go on top of the file:

See: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Fixed